optimade/__init__.py

```diff
@@ -1,2 +1,10 @@
-__version__ = "0.8.1"
+# coding: utf-8
```

Comment:
No need for this line I guess?
Or is it best practice since it's a "root" `__init__.py` file?
```suggestion
```